src/applications/personalization/profile/components/personal-information/ContactInformationSection.jsx

```diff
@@ -8,18 +8,42 @@ import PhoneNumbersTable from './phone-numbers/PhoneNumbersTable';
 
 const ContactInformationSection = ({ className }) => (
   <div className={className}>
+    <div className="vads-u-margin-bottom--2">
```

Comment:
Sadly we can't pass classed directly to the AdditionalInfo component so we need to use a wrapper div